components/authz-service/server/v2/project.go

```diff
@@ -108,6 +110,9 @@ func (s *state) CreateProject(ctx context.Context,
 	if err != nil {
 		if err == storage_errors.ErrConflict {
 			return nil, status.Errorf(codes.AlreadyExists, "project with ID %q already exists", req.Id)
+		} else if err == storage_errors.ErrMaxProjectsAllowed {
+			return nil, status.Errorf(codes.FailedPrecondition,
```

Comment:
My other comment seems to have disappeared, but I think "max of %d projects exceeded while IAM v2 Beta" speaks more to the error over the state

Reply:
i was thinking FailedPrecondition was not quite the right error code, but i couldn't find anything better. 

i did find this though: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/418 😂

Reply:
I don't have an opinion. @susanev I leave the final copywriting to you.

Reply:
let's keep it how it is, the ui text is using the word _allowed_ so would like to align with that.